js/WebViewTypes.js

```diff
@@ -0,0 +1,332 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+import type {Node, Element} from 'react';
+import type {EdgeInsetsProp} from 'EdgeInsetsPropType';
+import type {ViewStyleProp} from 'StyleSheet';
+import type {ViewProps} from 'ViewPropTypes';
+
+export type WebViewErrorEvent = {
+  domain: any,
+  code: any,
+  description: any,
+};
+
+export type WebViewEvent = Object;
```

Comment:
It would be great to get this typed more accurately

Reply:
Had a look at the native code and made a more exact type looking at Android's `createWebViewEvent` and iOS RCTWebView's `baseEvent`